crates/rpc-types/src/eth/transaction/access_list.rs

```diff
@@ -48,6 +48,13 @@ impl AccessList {
     }
 }
 
+impl From<AccessList> for alloy_eips::eip2930::AccessList {
+    fn from(value: AccessList) -> Self {
+        // SAFETY: Same repr and size
```

Comment:
can we make a test that tests this safety assumption?

Reply:
this should no longer be necessary after #324

can take over #324 and push that through